### PR TITLE
Use `wp_safe_redirect()` for safer redirects

### PR DESCRIPTION
**Pull request by danielbachhuber**

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@rmccue #reviewmerge

File: lib/class-wp-rest-oauth1-ui.php [hunk 1]
```diff
@@ -166,7 +166,7 @@ public function handle_callback_redirect( $verifier ) {
 		$args = urlencode_deep( $args );
 		$callback = add_query_arg( $args, $callback );
 
-		wp_redirect( $callback );
+		wp_safe_redirect( $callback );
```
Comment (rmccue) [inline lib/class-wp-rest-oauth1-ui.php]: This one can't use safe redirection, as it's not on the site itself. All the rest look good. :+1:

Comment (danielbachhuber) [inline lib/class-wp-rest-oauth1-ui.php]: Updated ec0b91dbf283d80e8be71d051ba6f47dd2c16e2f

**Comment by rmccue** [comment]
Thanks :sparkles:

**Pull request merged**